assets/src/edit-story/elements/text/display.js

```diff
@@ -142,14 +128,21 @@ function TextDisplay({
   });
 
   if (backgroundTextMode === BACKGROUND_TEXT_MODE.HIGHLIGHT) {
+    const foregroundContent = content;
+    // Setting the text color of the entire block to black essentially removes all inline
+    // color styling allowing us to apply transparent to all of them.
+    const backgroundContent = useMemo(
+      () => getHTMLFormatters().setColor(content, createSolid(0, 0, 0)),
```

Comment:
Is the `createSolid` missing an alpha 0?

Reply:
No - we apply solid opaque black, which is default meaning that [_no color style_ will be applied to the entire text field](https://github.com/google/web-stories-wp/pull/1323/files#diff-7c25b43d56bb77d2a8190efa2f7e0a2cR81-R83). Thus allowing us to overwrite the whole thing with `color:transparent`.